converters/converter.py

```diff
@@ -13,7 +14,7 @@ class Converter(object):
     EXCLUDED_FILES = ["license.md", "manifest.json", "package.json", "project.json", 'readme.md']
 
     def __init__(self, source, resource, cdn_bucket=None, cdn_file=None, options=None):
-        self.logger = ConvertLogger()
+        self.log = ConvertLogger()
```

Comment:
FYI, this is a different logger than the built-in Python logger to log issues with a conversion, seen by the user on the live.door43.org page